R/ggplotly.R

```diff
@@ -587,6 +591,26 @@ gg2list <- function(p, width = NULL, height = NULL,
         title = axisTitleText,
         titlefont = text2font(axisTitle)
       )
+
+      non_default_side <- isTRUE(scales$get_scales(xy)[["position"]] != default_axis)
+      
+      ## Move axis and change anchor if necessary
+      if (has_facet(plot)) {
+        if (non_default_side) {
+          if (xy == "x") {
+            ## Facet labels are always on top, I hope???
+            axisObj[["ticklen"]] <- axisObj[["ticklen"]] + 
+              (unitConvert(stripText, "pixels", type) * 2.5)
+            if (nRows > 1) {
+              axisObj[["anchor"]] <- "y"
+            } 
```

Comment:
I think the if statement is actually redundant, since it will always need to be anchored to the first row, whether there is more than one row or not